theseus/labs/lie/lie_tensor.py

```diff
@@ -143,7 +159,8 @@ class LieTensor(_LieTensorBase):
         torch.Tensor.layout.__get__,  # type: ignore
         torch.Tensor.requires_grad.__get__,  # type: ignore
         torch.Tensor.retains_grad.__get__,  # type: ignore
-        torch.Tensor.shape.__get__,  # type: ignore
+        torch.Tensor.storage,  # type: ignore
```

Comment:
Couldn't figure out where this is used.

Reply:
When any of [these](https://github.com/facebookresearch/theseus/pull/494/files/a4e93e7d03f065d06f2849c4a678ae6190b49ad0#diff-640b323e767b7377419a10b17fa0b39e8bf056486fa0921a44f986eca932a8bfR151) functions is called by `torch`, our class simply runs the default method provided by `torch.Tensor` class. See [here](https://github.com/facebookresearch/theseus/pull/494/files/a4e93e7d03f065d06f2849c4a678ae6190b49ad0#diff-640b323e767b7377419a10b17fa0b39e8bf056486fa0921a44f986eca932a8bfR212).